### PR TITLE
[Playground] Gradle task to remove snippet by its ID

### LINKED ISSUE #26164: [Task]: Add a tool to remove Playground snippets by snippet ID
### What needs to happen?

There should be a way to remove already published snippets by their ID, similar to how we have tool to clean up old snippets

### Issue Priority

Priority: 3 (nice-to-have improvement)

### Issue Components

- [ ] Component: Python SDK
- [ ] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [ ] Component: IO connector
- [ ] Component: Beam examples
- [X] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

### PR DESCRIPTION
**Pull request by TSultanov**
Add Gradle task to allow remove snippets form datastore by the snippet ID

resolves #26164

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: playground/backend/cmd/remove_unused_snippets.go [hunk 1]
```diff
@@ -16,44 +16,130 @@
...
+	return nil
+}
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("expected 'cleanup' or 'remove' subcommands")
+		os.Exit(1)
+	}
+
+	fmt.Println("Removing unused snippets is running...")
```
Comment (olehborysevych) [inline playground/backend/cmd/remove_unused_snippets.go:79]: Please consider moving this below the place where we know if this is a cleanup or remove

File: playground/backend/cmd/remove_unused_snippets.go [hunk 2]
```diff
@@ -16,44 +16,130 @@
...
+
+		if *dayDiff < 0 {
+			fmt.Println("day_diff is required and should be greater than 0")
+			os.Exit(1)
+		}
+
+		logger.SetupLogger(context.Background(), cwd, *projectId)
+		err = cleanup(*dayDiff, *projectId)
+		if err != nil {
+			fmt.Printf("Couldn't cleanup the database, err: %s \n", err.Error())
```
Comment (olehborysevych) [inline playground/backend/cmd/remove_unused_snippets.go:113]: consider using log.Fatal

**Review by olehborysevych — APPROVED** [review]
LGTM

File: playground/README.md [hunk 3]
```diff
@@ -150,6 +150,15 @@ cd beam
 ./gradlew playground:backend:removeUnusedSnippet -DdayDiff={int} -DprojectId={string}
 ```
 
+## Removing a specific snippet
+
+Run the method to remove a specific code snippet from the Cloud Datastore.
+
+```
+cd beam
+./gradlew playground:backend:removeSnippet -DsnippetId={string} -DprojectId={string}
```
Comment (rshamunov) [inline playground/README.md:159]: We need support for custom Datastore namespaces

**Review by rshamunov — APPROVED** [review]
LGTM

File: playground/backend/cmd/remove_unused_snippets.go [hunk 4]
```diff
@@ -16,46 +16,141 @@
...
+		os.Exit(1)
+	}
+
+	cwd, err := os.Getwd()
+	if err != nil {
+		fmt.Printf("Couldn't get the current working directory, err: %s \n", err.Error())
+		os.Exit(1)
+	}
+
+	switch os.Args[1] {
```
Comment (damccorm) [inline playground/backend/cmd/remove_unused_snippets.go:98]: Should we have a default case to catch if the user didn't correctly specify either cleanup or remove?

**Review by damccorm — COMMENTED** [review]
Just had one nit, otherwise LGTM

File: playground/backend/cmd/remove_unused_snippets.go (hunk 4 above)
Comment (TSultanov) [reply playground/backend/cmd/remove_unused_snippets.go:98]: Good catch, thanks!

**Review by damccorm — APPROVED** [review]
LGTM, thanks!

**Pull request merged**